stable/newrelic-infrastructure/templates/daemonset.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  labels: {{ include "newrelic-infra.labels" . | indent 4 }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+  name: {{ template "newrelic-infra.fullname" . }}
+spec:
+  template:
+    metadata:
+    {{- if .Values.podAnnotations }}
+      annotations:
+{{ toYaml .Values.podAnnotations | indent 8}}
+    {{- end }}
+      labels: {{ include "newrelic-infra.labels" . | indent 8 }}
```

Comment:
Usually only `app` and `release` labels are added here.

Reply:
as above